### ISSUE TITLE
Save data in browser

### ISSUE THREAD

**Issue by HishamAlmannai** [HishamAlmannai/capstone-project#10]
## Value Proposition
**As a** User
**I need to** keep my data after refreshing the page
**to be able to** safely add and edit data over a longer period of time.
## Description
![Bildschirmfoto](https://user-images.githubusercontent.com/101577601/166927734-00a079b9-9465-4cb1-83ac-918fe8e275ff.png)
## Acceptance Criteria
- [x]  The Check-boxes need to stay in the status after browser refresh and restart
- [ ] ~~The edited text needs to stay modified after browser refresh and restart~~ Edit not implemented at the moment.
- [x] The deleted task needs to stay removed after browser refresh and restart
## Tasks
- [x] Create a new branch "save-data-in-browser"
- [x] Implement persist
- [ ] ~~Add test to check if checkbox state is saved~~ Not responsible for test of feature Localstorage 
- [ ] ~~Add test to check if edited data is saved~~ Not responsible for test of feature Localstorage 
- [ ] ~~Add test to check if deleted data is removed~~ Not responsible for test of feature Localstorage 
## Complexity
M

**Comment by PReichetanz** [comment]
The user does not know that we use the browser to save data, so I would rephrase the value proposition:
```
As a user
I need to keep my data after refreshing the page 
to be able to safely add and edit data over a longer period of time.
```

**Comment by HishamAlmannai** [comment]
Thanks for the comment & proposal.
